### DISCUSSION TITLE
Routing problem: 404 NOT FOUND when reloading pages other than index page

### DISCUSSION THREAD

**Discussion by piavgh** [vercel/vercel#3985] [General] [answered]
Recently I bought a Next.js template on Themeforest and I tried to deploy it to Zeit Now.

This is a monorepo that using Lerna and Yarn workspace. Code of the Next.js app is inside `packages/landing` folder

Everything is fine with the index page (the page at `/` route), I can reload it without any problem.

The problem arises when I tried to add a new page inside `pages` folder. For example `/privacy-policy` or `/terms-of-service`, ...

If I navigate the these pages through a `<Link />` from the index page, it works fine. But if I tried to reload these pages or send my users a direct link to them, they can't access these pages. Instead, my users will see 404 NOT FOUND.

You can see what I mean by going to this page https://superprops-2-r09pdhfab.now.sh/. It works fine when reloading.

But if you open this page https://superprops-2-r09pdhfab.now.sh/saas, you will see 404 (the code for `/saas` is also deployed)

In my localhost development, it works fine for all pages when reloading. So I think that the problem is from Zeit Now configuration.

This is the code inside `now.json`:

```json
{
  "version": 2,
  "builds": [
    { "src": "packages/landing/package.json", "use": "@now/static-build" }
  ],
  "routes": [
    {
      "src": "/(.*)",
      "dest": "/packages/landing/$1",
      "headers": {
        "x-request-path": "$1"
      }
    }
  ]
}

```

How to fix this? Thank you.

**Comment by piavgh** [comment]
Also when I tried to export this project and upload to Netlify for testing purpose, it works fine. So I think it's not a problem with the code.

**Comment by paulogdm** [comment]
Can you check if the "output folder" has your page there? The route looks correct as far as I can see.

**Reply by piavgh** [reply]
as I can see, there is no output folder.
You can check my deployment here https://zeit.co/hoangtrinhj/superprops-2/b66v75tes/source

**Reply by piavgh** [reply]
by the way, this is the reason why in my other discussion (https://github.com/zeit/now/discussions/3987#discussioncomment-2547), I ask about @now/next, @now/static-build.

At the moment, I guess I have to refactor the template's code (don't use monorepo anymore, instead switch to zero config folder structure)

**Comment by piavgh** [comment, marked as answer]
I create a new Next project and move all code into this new project and it works fine zero-config.
So I guess problem is solved
